k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -733,6 +759,35 @@ function deployCortxConfigMap()
         --from-file="${auto_gen_ha_path}"
 }
 
+# https://stackoverflow.com/a/26665585/508923
+function choose()
+{
+    echo "${1:RANDOM%${#1}:1}" "${RANDOM}";
+}
+
+function choosealpha()
+{
+    choose '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
+}
+
+function pwgen()
+{
+    local first
+    local rest
+    first=$(choosealpha | awk '{printf "%s", $1}')
+    rest=$({
+        choose '!@#$%^'
+        choose '0123456789'
+        choose 'abcdefghijklmnopqrstuvwxyz'
+        choose 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+        for i in $( seq 1 $(( 4 + RANDOM % 8 )) )
```

Comment:
Is this choosing a random password length? If so, my question is, why? Just pick something that's reasonably secure, like 16? Minimum should be at least 8. Choosing a stronger default is better.

Reply:
Good point... I'm going with 16 char now.